### ISSUE TITLE
how to define it to multiclass secmantic segmentation?

### ISSUE THREAD

**Issue by Adithia99** [zhixuhao/unet#180]
how to define it to multiclass? ,  i was tried to change last layer to softmax, loss function to categoricall_crosentropy but the result still binary (just white and black) , i know we must define first in "data" codes, (example : background[0,0,0],car[0,0,255],object[255,0,0], but i dont know how to write it..

please help me i use according this "data" codes thank you
